mono/metadata/class-internals.h

```diff
@@ -1069,7 +1052,8 @@ mono_metadata_has_generic_params (MonoImage *image, guint32 token);
 
 MONO_API MonoGenericContainer *
 mono_metadata_load_generic_params (MonoImage *image, guint32 token,
```

Comment:
This function is MONO_API. Do you know the reason? Did you check whether the SDKs use it?

Reply:
git grep in XA, XM/XI, VSfM didn't find anything.